### DISCUSSION TITLE
Vue folder not getting generated on ignite scaffold chain github.com/alice/checkers

### DISCUSSION THREAD

**Discussion by rajeshsatpathy1** [cosmos/community#11] [Clients (JS)] [answered]
Hello everyone,

The Vue folder does not get generated as mentioned in the tutorial (https://tutorials.cosmos.network/hands-on-exercise). I had to scratch my head a lot and later luckily solved it by coming across scaffolding vue on the app. I followed the documentation at https://docs.ignite.com/clients/vue.

Is this an expected behavior? Or does the tutorial need to be updated for this?

**Comment by julienrbrt** [comment]
Hi, what is the version of Ignite that you were using and which tutorial were you following (your link gives 404)?

**Reply by rajeshsatpathy1** [reply]
Hi Julien,
Sorry for that. Can you check with this link https://tutorials.cosmos.network/hands-on-exercise/1-ignite-cli/1-ignitecli.html#your-chain.
My version of ignite is v0.26.1

The rest of the info from ignite version is as follows:
Ignite CLI version: v0.26.1
Ignite CLI config version: v1
Cosmos SDK version: v0.46.7
Your OS:	linux
Your arch: amd64
Your Node.js version: v18.15.0
Your go version: go version go1.20.2 linux/amd64
Is on Gitpod: false

**Reply by julienrbrt** [reply, marked as answer]
Alright, the tutorial is based on ignite version v0.22.1 it looks like.
This means the content has been tested with that ignite version. Possibly, the frontend is not generated anymore in later versions.

**Reply by rajeshsatpathy1** [reply]
Yes that seemed to be the issue. I checked with the v0.22.1 version and the vue folder is getting generated as expected.

Thank you!
